lms/static/sass/_developer.scss

```diff
@@ -140,79 +191,39 @@
       width: 1px;
     }
 
-    .pagination-form,
-    .current-page,
-    .page-divider,
-    .total-pages {
-      display: inline-block;
-    }
-
-    .current-page,
-    .page-number-input,
-    .total-pages {
-      @extend %t-copy-base;
-      @extend %t-strong;
-      width: ($baseline*2.5);
-      margin: 0 ($baseline*0.75);
-      padding: ($baseline/4);
-      text-align: center;
-      color: $gray;
-    }
-
-    .current-page {
-      @extend %ui-depth1;
-      position: absolute;
-      @include left(-($baseline/4));
-    }
-
-    .page-divider {
-      @extend %t-title4;
-      @extend %t-regular;
-      vertical-align: middle;
-      color: $gray-l2;
-    }
+    .page-number-input {
+      @include transition(all $tmg-f2 ease-in-out 0s);
+      border: 1px solid transparent;
+      border-bottom: 1px dotted $gray-l2;
+      border-radius: 0;
+      box-shadow: none;
+      background: none;
 
-    .pagination-form {
-      @extend %ui-depth2;
-      position: relative;
-
-      .page-number-label,
-      .submit-pagination-form {
-        /* This wasn't working for me, so I directly copied the rule
-        @extend %cont-text-sr; */
-        border: 0;
-        clip: rect(0 0 0 0);
-        height: 1px;
-        margin: -1px;
-        overflow: hidden;
-        padding: 0;
-        position: absolute;
-        width: 1px;
+      &:hover {
+        background-color: $white;
+        opacity: 0.6;
       }
 
-      .page-number-input {
-        @include transition(all $tmg-f2 ease-in-out 0s);
+      &:focus {
+        // borrowing the base input focus styles to match overall app
+        @include linear-gradient($paleYellow, tint($paleYellow, 90%));
+        opacity: 1.0;
+        box-shadow: 0 0 3px $shadow-d1 inset;
+        background-color: $white;
         border: 1px solid transparent;
-        border-bottom: 1px dotted $gray-l2;
-        border-radius: 0;
-        box-shadow: none;
-        background: none;
-
-        &:hover {
-          background-color: $white;
-          opacity: 0.6;
-        }
-
-        &:focus {
-          // borrowing the base input focus styles to match overall app
-          @include linear-gradient($paleYellow, tint($paleYellow, 90%));
-          opacity: 1.0;
-          box-shadow: 0 0 3px $shadow-d1 inset;
-          background-color: $white;
-          border: 1px solid transparent;
-          border-radius: 3px;
-        }
+        border-radius: 3px;
       }
     }
   }
 }
+
+// moved here from _teams.scss
+.search-tools {
+  @extend %t-copy-sub1;
+  margin: ($baseline/10) $baseline;
+  color: $gray-l1;
+
+  p, span { // override
+    color: inherit;
+  }
+}
```

Comment:
@frrrances We copied this sass to get some of things to style correctly on the bookmarks list. Is this needed or can we include this some other way? A sandbox is linked if you would like to review the changes. Thanks.